recipes-robot/fs-overlay/files/setup-partition.sh

```diff
@@ -0,0 +1,21 @@
+if ! test -e /dev/mmcblk0p4
+then
+	# create partition using sfdisk
+	echo 'size=+, type=L' | sfdisk -N 4 /dev/mmcblk0
```

Comment:
If we use a script like this to create this partition at runtime, then the first boot would take a pretty long time and be non-interactive, right? It's also sort of hard to read prompted sfdisk like this.

I think it might be better for this last partition to be established in tezi. I guess this would be an ok backup, but especially right now I'd rather it fail hard if that partition doesn't exist.